emr-dynamodb-hadoop/pom.xml

```diff
@@ -89,6 +94,13 @@
                     <maxAllowedViolations>0</maxAllowedViolations>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-assembly-plugin</artifactId>
```

Comment:
Why are we adding this plugin?

Reply:
Add this plugin to package the dependency (like AWS SDK 2.x) into the JAR file.